doc/ug/getting_started_dv.md

```diff
@@ -61,28 +59,27 @@ There is support for Cadence Xcelium as well, which is being slowly ramped up.
 
 ## Building and Running Tests
 
-The `uvmdvgen` tool provides an empty shell sequence at `dv/env/seq_lib/<ip>_sanity_vseq.sv` for developing the sanity test.
-The sanity test can be run as-is by invoking `make`, as a "hello world" step to bring the DUT out of reset.
+The `uvmdvgen` tool provides an empty shell sequence at `dv/env/seq_lib/<dut>_sanity_vseq.sv` for developing the sanity test.
+The sanity test can be run as-is by invoking `dvsim.py`, as a "hello world" step to bring the DUT out of reset.
 ```console
 $ cd path-to-dv
-$ make [SIMULATOR=xcelium] [WAVES=1]
+$ util/dvsim/dvsim.py <dut>_sim_cfg.hjson -i <dut>_sanity [--waves] [--tool xcelium]
```

Comment:
thanks for the suggestion. I originally thought since line 65 already navigate users to the dv folder, they should be able to find the hjson file in the same folder. But then the `util/dvsim/dvsim.py` path would be invalid. I will delete line 65 as well.